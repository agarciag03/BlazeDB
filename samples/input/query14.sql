SELECT * FROM Student, Enrolled WHERE Student.A = Enrolled.A AND Student.B = 200 AND Student.A = 1;