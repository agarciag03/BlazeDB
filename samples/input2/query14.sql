SELECT * FROM Student, Enrolled WHERE Student.A = Enrolled.A AND Student.B = 200 AND Enrolled.A = 1;