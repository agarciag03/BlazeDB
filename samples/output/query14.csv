1, 200, 50, 33, 1, 101, 75
1, 200, 50, 33, 1, 102, 82
1, 200, 50, 33, 1, 103, 92
