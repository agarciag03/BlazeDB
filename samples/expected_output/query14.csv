1, 200, 50, 33, 1, 103, 92
1, 200, 50, 33, 1, 102, 82
1, 200, 50, 33, 1, 101, 75